CRM/Contact/Import/Form/DataSource.php

```diff
@@ -185,7 +185,6 @@ public function postProcess() {
       'onDuplicate' => $this->getSubmittedValue('onDuplicate'),
       'dedupe' => $this->getSubmittedValue('dedupe_rule_id'),
       'contactType' => $this->getSubmittedValue('contactType'),
-      'contactSubType' => $this->getSubmittedValue('subType'),
```

Comment:
This array is the values being 'set' on the form - it can more or less go altogether as we access them through `$this->getSubmittedValue` in most places in the flow now